packages/bodiless-demo/src/shadow/@bodiless/vital-templates/GenericTemplate.ts

```diff
@@ -26,6 +28,11 @@ const ContentListing = asGenericTemplateToken(
   {
     Components: {
       ContentWrapper: withPrependChild(Description, 'Description'),
+      TopContent: as(
+        withDesign({
+          HeroCard: on(CardStatic)(vitalCardShadowed.Basic),
```

Comment:
why do we need to refer to vitalCardShadowed explicitly here?  We should just be able to use vitalCard, no?

Reply:
Address by moving the custom token to src/components.